documentation/docs/identus/mediator.md

```diff
@@ -1,20 +1,20 @@
 
 ## Mediator's Role in Decentralized Identity Transactions
 
 Within a decentralized identity framework, the Mediator bridges the [Holder's](/docs/concepts/glossary#holder) device and other key players, such as [Issuers](/docs/concepts/glossary#issuer) and [Verifiers](/docs/concepts/glossary#verifier). For instance, if a Holder wishes to obtain a credential from an Issuer, the Mediator can [relay](/docs/concepts/glossary#relay) or mediate this request, ensuring smooth interaction between both parties. Similarly, when a holder must present a credential to a verifier, the Mediator steps in again to mediate or relay the transaction. By leveraging a Mediator, identity wallets can offer secure and reliable communication between all involved entities. This approach also empowers holders to maintain control over their data and identities.
 
 ## Security, Scalability, and Efficiency
 
-The Mediator is an integral part of Atala PRISM, facilitating secure communication without the need for a centralized authority. It seamlessly connects various entities in the PRISM ecosystem, including Holders, Issuers, and Verifiers, ensuring the system remains secure, scalable, and efficient.
+The Mediator is an integral part of Identus, facilitating secure communication without the need for a centralized authority. It seamlessly connects various entities in the Identus ecosystem, including Holders, Issuers, and Verifiers, ensuring the system remains secure, scalable, and efficient.
 
 ## Technical Specifications
 
-The PRISM Mediator is an open-source initiative. For more details, you can refer to the [atala-prism-mediator Github repository](https://github.com/input-output-hk/atala-prism-mediator). The project aims to address the inherent limitations of edge agents, which are not always online, by assuming that the Mediator is perpetually online. Employing DIDComm v2 protocols and cryptographic methods, the Mediator guarantees all exchanged messages' authenticity, integrity, and confidentiality.
+The Mediator is an open-source initiative. For more details, you can refer to the [atala-prism-mediator Github repository](https://github.com/input-output-hk/identus-mediator). The project aims to address the inherent limitations of edge agents, which are not always online, by assuming that the Mediator is perpetually online. Employing DIDComm v2 protocols and cryptographic methods, the Mediator guarantees all exchanged messages' authenticity, integrity, and confidentiality.
```

Comment:
This link cannot be changed before the migration is done as this repo does not exist yet. We will do that after the migration is done.

Reply:
reverted

Reply:
@CryptoKnightIOG , maybe I do not know where to look but I do not see that the link is reverted yet. Can you double check?